src/components/about/sections/ManagementBoardSection/ManagementBoardDescription.tsx

```diff
@@ -0,0 +1,23 @@
+import { Button } from '@mui/material'
+import { Description } from 'components/about/AboutPage.styled'
+import { useTranslation } from 'next-i18next'
+import { useState } from 'react'
+
+export function ManagementBoardDescritpion({ description }) {
+  const { t } = useTranslation()
+
+  const [show, setShow] = useState(false)
+
+  return (
+    <>
+      <Description variant="body2" height={show ? 'auto' : '200px'}>
+        {description}
+      </Description>
```

Comment:
Cutting off elements like that is not particularly good. You might cut off text in the middle of it's y axis and it will look bad. 
You should consider a different approach truncating the text to ex. 200 symbols and then adding .... at the end!

Reply:
Thanks I think you are right, I was thinking that there must be a better way and this seems way better

Reply:
> Cutting off elements like that is not particularly good. You might cut off text in the middle of it's y axis and it will look bad.
> You should consider a different approach truncating the text to ex. 200 symbols and then adding .... at the end!
> Here is a lodash function you can use https://lodash.com/docs/4.17.15#truncate

Maybe try this (CSS only fix)
/sorry can't really implement this myself yet as I still have to learn the tech behind, but I can write the CSS hack for someone that can do it accordingly/

<img width="1577" alt="Screenshot 2022-11-17 at 4 08 07" src="https://user-images.githubusercontent.com/118026471/202337497-a83f282b-c5d7-454e-81e1-c2722402b069.png">

*_There is a second issue of Member names not being of equal height (and vertically centered), thus shifting the paragraphs below when the Member names consist of more than one lines, which is also being fixed here_.

/add proposed styles to existing/

// fix titles
`h6.MuiTypography-subtitle1 {
  height: 3em;
  display: flex;
  flex-direction: column;
  justify-content: center;
}`

// truncate to 14 lines of text
`p.MuiTypography-body2 {
  max-height: calc(1em*14*1.43); 
  overflow: hidden;
  position: relative;
}`

// fade out the last 4 lines instead of adding an ellipsis (...)
`p.MuiTypography-body2:after {
  content: "";
  position: absolute;
  bottom: 0px;
  left: 0px;
  width: 100%;
  height: calc(0.875rem*1.43*4);
  background: linear-gradient(0deg, rgb(255, 255, 255) 0%, rgba(255,255,255,.33) 65%, rgba(255,255,255,0) 100%);
  z-index: 1;
}`

** Add a **_Learn more_** button below.